Client/Frontend/Browser/BrowserViewController/BrowserViewController+URLBarDelegate.swift

```diff
@@ -121,7 +121,12 @@ extension BrowserViewController: URLBarDelegate {
                     self.legacyShowSettings(deeplink: .contentBlocker)
                 }
             }
-
+            
+            if CoordinatorFlagManager.isEtpCoordinatorEnabled {
+                
```

Comment:
nit, remove extra space
```suggestion
            if CoordinatorFlagManager.isEtpCoordinatorEnabled {
```